dotnet/src/SemanticKernel.Abstractions/SemanticFunctions/PromptTemplateConfig.cs

```diff
@@ -107,69 +54,63 @@ public class InputConfig
     }
 
     /// <summary>
-    /// Schema - Not currently used.
+    /// AI service configuration
     /// </summary>
-    [JsonPropertyName("schema")]
-    [JsonPropertyOrder(1)]
-    public int Schema { get; set; } = 1;
+    public class ServiceConfig
+    {
+        /// <summary>
+        /// Model ID in AI service provider (e.g. text-davinci-003).
+        /// </summary>
+        [JsonPropertyName("model_id")]
+        [JsonPropertyOrder(1)]
+        public string ModelId { get; set; } = string.Empty;
```

Comment:
How are you mapping this value to Azure, where the API requires a deployment name, which is customized by developers?